Lib/test/test_heapq.py

```diff
@@ -9,7 +9,7 @@
 from operator import itemgetter
 
 py_heapq = support.import_fresh_module('heapq', blocked=['_heapq'])
-c_heapq = support.import_fresh_module('heapq', fresh=['_heapq'])
+c_heapq = None
```

Comment:
Is this necessary to change? I feel like I've seen these working in other test files, and a benefit of leaving the `import_fresh_module` is that the test file doesn't require any changes once the native module is implemented.

Reply:
Indeed, this actually doesn't raise. Thanks for the feedback, will get to it now!